src/constructs/iam/policies/ssm.ts

```diff
@@ -33,7 +33,7 @@ export class GuSSMRunCommandPolicy extends GuPolicy {
     };
   }
 
-  constructor(scope: GuStack, id: string, props: GuPolicyProps) {
+  constructor(scope: GuStack, id: string = "SSMRunCommandPolicy", props: GuPolicyProps = { overrideId: false }) {
```

Comment:
I'm not sure that I like defining a default object for the prop and then also having a `getDefaultProps` function. It makes it a bit confusing to have two different places to define default props. If there are no required props, maybe the whole props parameter should be optional instead? 

```suggestion
  constructor(scope: GuStack, id: string = "SSMRunCommandPolicy", props?: GuPolicyProps) {
```

Reply:
Yes, I think I agree with this, it gets us close to same shape as CDK classes too.